server/src/internalClusterTest/java/org/opensearch/search/basic/SearchWithRandomExceptionsIT.java

```diff
@@ -193,6 +193,11 @@ public void testRandomExceptions() throws IOException, InterruptedException, Exe
                 logger.info("expected SearchPhaseException: [{}]", ex.getMessage());
             }
         }
+
+        // as the index refresh may fail, so the translog in the index will be not flushed,
+        // and `TranslogWriter.buffer` is not null, which causes arrays not been released,
+        // so we need to close the index to release the arrays.
+        cluster().wipeIndices("test");
```

Comment:
The cleanup is not supposed to be called manually, `OpenSearchTestClusterRule::after` is designed to clean up the indices after each test method call

Reply:
There has two `after`, `OpenSearchTestCase::after` and  `OpenSearchTestClusterRule::after`.

- `OpenSearchTestCase::after` will call `checkStaticState` to check whether the arrays are released.
- `OpenSearchTestClusterRule::after` will clean up the indices.

we can debug and find that `OpenSearchTestCase::after` is called before `OpenSearchTestClusterRule::after`.

Reply:
> we can debug and find that OpenSearchTestCase::after is called before OpenSearchTestClusterRule::after.

This is correct, thank you @kkewwei